libcrux-ml-kem/c/libcrux_mlkem_avx2.c

```diff
@@ -2534,11 +2613,10 @@ with const generics
 static inline libcrux_ml_kem_polynomial_PolynomialRingElement_d2 clone_d5_48(
     libcrux_ml_kem_polynomial_PolynomialRingElement_d2 *self) {
   libcrux_ml_kem_polynomial_PolynomialRingElement_d2 lit;
-  core_core_arch_x86___m256i ret[16U];
+  __m256i ret[16U];
```

Comment:
Could this collide with the real `__m256i` type?

Reply:
right but there is a typedef anyhow https://github.com/cryspen/libcrux/blob/main/libcrux-ml-kem/c/intrinsics/libcrux_intrinsics_avx2.h#L20 so that's cool